core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -286,7 +286,7 @@ public static Builder builder(Session defaultSession)
         return new Builder(defaultSession);
     }
 
-    private LocalQueryRunner(
+    private static LocalQueryRunner create(
```

Comment:
do not add new overrides, just add `Set<SystemSessionPropertiesProvider> systemSessionProperties` argument